types/koa__router/index.d.ts

```diff
@@ -156,6 +156,7 @@
 }
 
 declare class Router<StateT = Koa.DefaultState, ContextT = Koa.DefaultContext> {
+    opts: Router.RouterOptions;
```

Comment:
`methods` can be added too.

https://github.com/koajs/router/blob/master/lib/router.js#L53-L62